source/robot_model/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 set(LIBRARY_NAME robot_model)
 
-set(PINOCCHIO_VERSION 2.6.9)
-set(OSQP_EIGEN_VERSION 0.6.4)
+set(PINOCCHIO_VERSION 2.9.0)
+set(OSQP_EIGEN_VERSION 0.8.1)
 find_package(pinocchio ${PINOCCHIO_VERSION} REQUIRED)
 find_package(OsqpEigen ${OSQP_EIGEN_VERSION} REQUIRED)
```

Comment:
```suggestion
set(PINOCCHIO_VERSION 2.9.0)
set(FCL_HPP_VERSION 1.81.)
set(OSQP_EIGEN_VERSION 0.8.1)
find_package(pinocchio ${PINOCCHIO_VERSION} REQUIRED)
find_package(fcl-hpp ${FCL_HPP_VERSION} REQUIRED)
find_package(OsqpEigen ${OSQP_EIGEN_VERSION} REQUIRED)
```

Reply:
It requires minor changes, but yes indeed that part was missing.

Just two small notes, the name of the library is hpp-fcl and the version is 1.8.1.